book/02-git-basics/sections/tagging.asc

```diff
@@ -211,13 +211,25 @@ Now, when someone else clones or pulls from your repository, they will get all y
 
 ==== Checking out Tags
 
-You can't really check out a tag in Git, since they can't be moved around.
-If you want to put a version of your repository in your working directory that looks like a specific tag, you can create a new branch at a specific tag with  `git checkout -b [branchname] [tagname]`:
+You can check out a tag through the `git checkout` command the same way you would with a commit:
```

Comment:
So technically (and according to the output below) this puts you in "detached HEAD" state, which can be pretty confusing for complete newbies. We haven't even introduced the concept of HEAD, or the fact that branch pointers move when you make a commit. This might be too much information for the typical reader of this section.

Maybe something like this would be more appropriate:

> If you want to view the versions of files a tag is pointing to, you can do a `git checkout`, though this puts your repository into a state that isn't very useful:
>
> _[new example]_
>
> However, if you make changes and then create a commit, the tag will stay the same, and your commit won't be reachable from any branch. If you need to make changes, say if you're fixing a bug on an older version, you'll want to create a branch:
>
> _[old example]_

Reply:
Ah, that's a very good point; I hadn't considered that at all. I agree with your suggestion, though I think the wording of it could benefit from a slight tweak to emphasise that the second paragraph talks about the detached HEAD state. Something more like:

>If you want to view the versions of files a tag is pointing to, you can do a `git checkout`, though this puts your repository in "detached HEAD" state, which has some ill side effects:
>
>*[new example]*
>
>In "detached HEAD" state, if you make changes and then create a commit, the tag will stay the same, but your new commit won't belong to any branch and will be unreachable, except for by the exact commit hash. Thus, if you need to make changes—say you're fixing a bug on an older version, for instance—you will generally want to create a branch:
>
>*[old example]*

(Also, I have to admit I'm a bit fuzzy on the exact effects of detached HEAD, so if anything above is wrong, please do not hesitate to correct me.)

Reply:
Nope, that sounds perfect. Make that change and we can get this merged!